src/styles/_ProductDetailOrderForm.scss

```diff
@@ -72,9 +81,9 @@
   
    .details-order-form__faded-btn {
     width: 40%;
-    height: 3rem;
-    background-color: rgba(0, 0, 0, 0.4);
-    color: white;
+    @include m.button-style;
```

Comment:
I like the uniformity of the buttons. 
Could remove border to make more uniform, but I'm happy either way.